packages/core/src/VDocument.ts

```diff
@@ -156,4 +157,57 @@ export class VDocument {
             }
         }
     }
+    /**
+     * Insert a list at range.
+     *
+     * @param type
+     */
+    list(type: 'ordered' | 'unordered'): void {
```

Comment:
I *think* I would have created an enum for the type to avoid having to rewrite the possible types at multiple places, like you had to do in the `ListNode` definition and here.